src/lib/get-device-info.js

```diff
@@ -17,6 +17,22 @@ const getBrowserHeight = () => window.innerHeight;
 
 const getBrowserWidth = () => window.innerWidth;
 
+const getScreenHeight = () => {
+    try {
+        return window.screen.height;
+    } catch (err) {
+        return null;
+    }
```

Comment:
could we import `lodash` and then just do `_.get(window, 'screen.height', null)` (might be wrong, I forgot the exact API)

Reply:
Client doesn't currently include `lodash` unfortunately. We only have one dependency atm. I think the goal is to keep things as slim as possible.